### PR TITLE
[TSC]: Build with readthedocs

### PR DESCRIPTION
**Pull request by CsatariGergely**
This pr adds changes needed to build CNTT on readthedocs.org. 
Please note that this is still work in progress. 

Planned changes in this pr: 
- Move build specific files to a specific folder
- Move the sphinx extensions to a separate folder

Planned changes in other pr-s needed to be fixed:
- Table 5-7
- Table A-1
- Table 3-4
- Link to ../../../_sources/ref_cert/RC2/chapters/chapter04.md.txt
- Link to ../_sources/ref_model/README.md.txt
- Link to ../_sources/ref_arch/abbreviations.md.txt
- Link to ../../_sources/ven_impl/ven_template/README.md.txt
- Link to ../../_sources/tech/release_notes/README.md.txt
- Link to ../_sources/ref_model/abbreviations.md.txt
- [9.5 CNTT Field Trial Approach] (#9.5) 
- gov/README.html ToC
- RA1 Ch2 #53-consolidated-set-of-apis -> #5.3
- RA1 Ch2 `<!-- **Backlog of Req**<ol class="simple"> `
- RA1 Ch4
- RI1 Ch3.8
- RI1 Ch5.4.1
- RI1 Ch9.2.1
- ven_impl/ven_template/

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@CsatariGergely let me know once you need this to be merged.

**Comment by bfcohen** [comment]
Looks great!  Much needed.

File: doc/ref_arch/openstack/chapters/chapter04.md [hunk 1]
```diff
@@ -627,7 +625,7 @@ Containers are lightweight compared to Virtual Machines and leads to efficient r
 
 In Chapter 3, [Figure 3.2](../figures/RA1-Ch03-OpenStack-Services-Topology.png) shows a high level Virtualised OpenStack services topology. The containerized OpenStack services topology version is shown in Figure 4-7.
 
-<p align="center"><img src="../figures/RA1-Ch04-Containerised-OpenStack-Services-Stack.png" alt="Containerised OpenStack Services Topology"></br>Figure 4-7: Containerised OpenStack Services Topology.</p>
+<p align="center"><img src="../figures/RA1-Ch04-Containerised-OpenStack-Services-Stack.png" alt="Containerised OpenStack Services Topology"><br>Figure 4-7: Containerised OpenStack Services Topology.</p>
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter04.md:628]:
```suggestion
<p align="center"><img src="../figures/RA1-Ch04-Containerised-OpenStack-Services-Stack.png" alt="Containerised OpenStack Services Topology"><b>Figure 4-7: Containerised OpenStack Services Topology.</b></p>
```

**Comment by CsatariGergely** [comment]
Thanks. Here is the first build: https://cntt.readthedocs.io/en/latest/

**Pull request merged**